src/main/scala/io/shiftleft/js2cpg/preprocessing/TypescriptTranspiler.scala

```diff
@@ -57,6 +66,24 @@ class TypescriptTranspiler(override val config: Config, override val projectPath
     }
   }
 
+  private def removeComments(json: String): String = {
+    val mapper = new ObjectMapper
+    mapper.enable(JsonParser.Feature.ALLOW_COMMENTS)
+    mapper.writeValueAsString(mapper.readTree(json))
```

Comment:
Oh cool, you found a JSON Parser that understands comments. In `TsConfigJsonParser` we shell out to `tsc --showConfig` in order to drop those. Probably makes sense to switch that over to also normalize the JSON like this.

Reply:
Sounds reasonable. But that should be a separate PR then.

Reply:
Agreed, no reason to block this PR on that unrelated change.